src/app/(auth)/styles.tsx

```diff
@@ -42,17 +47,16 @@ export const PCentered = styled(P)`
   text-align: center;
 `;
 
-export const AuthHeading = styled.h1`
-  color: ${COLORS.blueDarker};
-  font-size: 48px;
+export const AuthHeading = styled(H1)`
+  font-size: 3rem;
```

Comment:
If this is already a styled `H1`, it will inherit the font size of 3 rem, so we can remove this line.